tests/unit/test_progress.py

```diff
@@ -10,6 +11,14 @@ def test_quiet(caplog, capsys):
         assert out_err.out == ""
         assert out_err.err == ""
     with caplog.at_level(logging.INFO, logger="dvc"):
+        for _ in Tqdm(range(10)):
+            pass
+        out_err = capsys.readouterr()
+        assert out_err.out == ""
+        assert out_err.err == ""
```

Comment:
So it's disabled by default in tests? This means critical test above doesn't test anything now?

Reply:
well it's disabled by default if not `isatty` which would include all tests :)

Reply:
Yes, to force it working in tests we will need the env var above )